sdk/java-sdk-spring/src/main/scala/kalix/javasdk/impl/ComponentDescriptorFactory.scala

```diff
@@ -48,19 +41,13 @@ import kalix.javasdk.impl.reflection.CombinedSubscriptionServiceMethod
 import kalix.javasdk.impl.reflection.KalixMethod
 import kalix.javasdk.impl.reflection.NameGenerator
 import kalix.javasdk.valueentity.ValueEntity
+import kalix.javasdk.view.View
 // TODO: abstract away spring dependency
+import kalix.javasdk.impl.Reflect.Syntax._
 import org.springframework.core.annotation.AnnotatedElementUtils
 import org.springframework.web.bind.annotation.RequestMapping
-
 private[impl] object ComponentDescriptorFactory {
 
-  implicit class MethodOps(javaMethod: Method) {
-    def isPublic = Modifier.isPublic(javaMethod.getModifiers)
-
-    def hasAnnotation[A <: Annotation](implicit ev: ClassTag[A]) =
-      javaMethod.getAnnotation(ev.runtimeClass.asInstanceOf[Class[Annotation]]) != null
-  }
```

Comment:
Took the opportunity to do some clean-up. This moved to `Reflect.Syntax` and I also added a few more extenions